### PR TITLE
Update draft-ietf-oauth-transaction-tokens.md

### PR DESCRIPTION
**Pull request by gffletch**
Addresses text regarding the `aud` claim throughout the document. Issue #76

### CODE REVIEW

**Comment by tulshi** [comment]
Although your changes sufficiently capture what is needed, what I liked about the previous draft is that it clearly identified that each Trust Domain MUST have a unique identifier, which is set as the `aud` value. I wonder if we can say that somewhere (outside of the Terminology section)

**Comment by gffletch** [comment]
Ok, let me see if there is a good place to add that. I wonder if it will be easier to merge PR #90 first and then let me fix this one as there may be conflicts?

**Comment by tulshi** [comment]
Makes sense to work on this after we merge #90

File: draft-ietf-oauth-transaction-tokens.md [hunk 1]
```diff
@@ -339,7 +339,7 @@ JWT claims as well as defines new claims. These claims are described below:
 : REQUIRED A unique transaction identifier as defined in Section 2.2 of {{RFC8417}}. When used in the transaction token, it identifies the entire call chain.
 
 `sub`:
-: REQUIRED A unique identifier for the subject as defined by the `aud` trust domain. Unlike OpenID Connect, the `sub` claim is NOT associated with the `iss` claim.
+: REQUIRED A unique identifier for the subject within the context of the `aud` trust boundary. Unlike OpenID Connect, the `sub` claim is NOT associated with the `iss` claim.
```
Comment (tulshi) [inline draft-ietf-oauth-transaction-tokens.md:342]: Did you intend for "trust boundary" to be something separaet from "trust domain"? I'd rather not invoke an additional concept if we can avoid it.

Comment (gffletch) [reply draft-ietf-oauth-transaction-tokens.md:342]: Yes, I agree. Let me fix that.

Comment (gffletch) [reply draft-ietf-oauth-transaction-tokens.md]: Updated

**Review by tulshi — APPROVED** [review]

**Pull request merged**